src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/HwfClaimNotificationService.java

```diff
@@ -96,7 +96,10 @@ public void logNotificationFailure(
     private Map<String, String> aggregateParams(Claim claim,
                                                 String submitterName) {
         ImmutableMap.Builder<String, String> parameters = new ImmutableMap.Builder<>();
-        parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
+
+        if (claim.getReferenceNumber() != null) {
```

Comment:
If there is no reference number generated yet, then the claim has not been issued yet. 
Also if we don't pass the placeholder value to the template, will notification still work or throw error?